pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -67,6 +67,11 @@ func (d *ProjectStatusDescriber) MakeGraph(namespace string) (osgraph.Graph, uti
 	if errs := parallel.Run(loadingFuncs...); len(errs) > 0 {
 		actualErrors := []error{}
 		for _, err := range errs {
+			// ignore not found errors for resources which may be disabled server side
+			// this check should be more specific but nothing is reported in the details.
+			if kapierrors.IsNotFound(err) {
+				continue
```

Comment:
@deads2k opinion on how to handle graph requests for types the server doesn't have enabled?